Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -656,6 +685,12 @@ lemma zero_def : (zero : W.Point) = 0 :=
   rfl
 #align weierstrass_curve.point.zero_def WeierstrassCurve.Affine.Point.zero_def
 
+lemma some_ne_zero {x y : R} (h : W.Nonsingular x y) : some h ≠ 0 := by rintro (_|_)
```

Comment:
I think there might a couple of places in the files where I implicitly use `some_ne_zero`, but via the `contradiction` tactic instead. Can you check that these have all been changed?

Reply:
It seems there are two appearances of `contradiction` in the `Affine` file; the appearance of `any_goals contradiction` in `map_injective` can be replaced but would make the proof longer, so I'd rather leave it as is.